README.md

```diff
@@ -137,7 +137,7 @@ docker run -d \
 --volume='readyset:/state' \
 --pull=always \
 -e DEPLOYMENT_ENV=quickstart_github \
-public.ecr.aws/readyset/readyset:beta-2023-06-29 \
+public.ecr.aws/readyset/readyset:latest \
```

Comment:
`latest` will pull the latest nightly build, not the actual latest _release_.  If we want the quickstart to always install the latest actual release, we'll need to apply some other tag (`latest-release`).  Our release cadence (and thus how we tag the images) is still in a bit of flux at the moment.

Reply:
Ah, good call. Ok, I can rework things to point for the latest release (public.ecr.aws/readyset/readyset:nightly-2023-07-31) for now.